src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -39,7 +41,7 @@ public void loadSomeData()
 					final Gson gson = new GsonBuilder().create();
 					final Map venue = gson.fromJson(content, Map.class);
 
-					final List<Map> placeCategories = (List<Map>) venue.get("categories");
+					placeCategories = (List<Map>) venue.get("categories");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make the enclosing method "static" or remove this set. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)